src/reducers/scrollAnchor.js

```diff
@@ -68,12 +68,23 @@ export function getScrollAnchor(state, newProps, oldProps) {
  *   changed: boolean,
  * }}
  */
-export function scrollTo(state, scrollY) {
+export function scrollTo(state, scrollY, isVerticalScrollExact) {
```

Comment:
There are calls to this function in other parts of the codebase where the third param is unspecified.
Some examples (but not limited to):
* https://github.com/schrodinger/fixed-data-table-2/blob/6bd7bb8e9d924b52a55f7eb4c0ea887221063051/src/reducers/index.js#L204
* https://github.com/schrodinger/fixed-data-table-2/blob/6bd7bb8e9d924b52a55f7eb4c0ea887221063051/src/actions/scrollActions.js#L59
* https://github.com/schrodinger/fixed-data-table-2/blob/6bd7bb8e9d924b52a55f7eb4c0ea887221063051/src/plugins/ScrollContainer.js#L31

This has the risk that some workflows in the table (eg: uncontrolled scrolling) don't use exact row offsets even when `isVerticalScrollExact` is true.

There's two approaches that I can immediately think of:

The first one is of course pass the prop `isVerticalScrollExact` in all these usages as the third param.
But this can be a bit cumbersome cause some of the calls are nested within components.

The second one -- which I'm more inclined towards -- is to simply copy the `isVerticalScrollExact` prop to the redux store as part of ["input" state](https://github.com/schrodinger/fixed-data-table-2/blob/6bd7bb8e9d924b52a55f7eb4c0ea887221063051/src/reducers/index.js#L48-L50) (see [`setStateFromProps()`](https://github.com/schrodinger/fixed-data-table-2/blob/6bd7bb8e9d924b52a55f7eb4c0ea887221063051/src/reducers/index.js#L246)) for similar cases.
Once this is done, this reducer can simply query `state.isVerticalScrollExact`, thus avoiding the need for the caller to pass it explicitly.

Reply:
Hello!

Thank you for this observation. I have modified as you suggested in order to avoid passing ```isVerticalScrollExact``` but instead pass it via state. Now the row heights are computed also for the ```DynamicRowHeightExample```. 

Do you think it's ok now?

Reply:
nice, that works.